lib/srv/usermgmt.go

```diff
@@ -288,7 +297,19 @@ func (u *HostUserManagement) createGroupIfNotExist(group string) error {
 	if err != nil && !isUnknownGroupError(err, group) {
 		return trace.Wrap(err)
 	}
-	err = u.backend.CreateGroup(group)
+	err = u.backend.CreateGroup(group, "")
+	if trace.IsAlreadyExists(err) {
+		return nil
+	}
+	return trace.Wrap(err)
+}
+
+func (u *HostUserManagement) createGroupWithGIDIfNotExist(gid string, groupname string) error {
+	_, err := u.backend.LookupGroupId(gid)
```

Comment:
Why do we look it up if we ignore the result? In which case it is required, where an error returned from `CreateGroup` wouldn't be sufficient?